help/user-guide/create-sample-isajson-compliant.md

```diff
@@ -3,59 +3,67 @@ title: SEEK User Guide - Create Sample in ISA-JSON compliant Experiments
 layout: page
 ---
 
-# Viewing samples in ISA-JSON compliant Experiments
-Samples in ISA-JSON compliant Experiments can be visualised in the "design" tab (Study or Assay design). Samples are grouped in tables: Study Sources table, Study Samples table, Assay Samples table, Experiment overview.
+# Working with Samples in ISA-JSON compliant Experiments
```

Comment:
In this page, viewing samples and creating samples are two different actions. Why do you suggest to do not keep the distinction?

Reply:
To me both fall under "Working with samples". The distinction is hence still there, I just felt an umbrella heading for the page was beneficial for the flow of information.

Reply:
ok